src/include/txn.h

```diff
@@ -135,10 +135,12 @@ struct __wt_txn_global {
 	wt_timestamp_t pinned_timestamp;
 	wt_timestamp_t recovery_timestamp;
 	wt_timestamp_t stable_timestamp;
+	wt_timestamp_t durable_timestamp;
 	bool has_commit_timestamp;
 	bool has_oldest_timestamp;
 	bool has_pinned_timestamp;
 	bool has_stable_timestamp;
+	bool has_durable_timestamp;
```

Comment:
Please alphabetize both of these new fields in their respective groupings.

Reply:
955effc